salt/kubernetes-minion/kubeconfig.jinja

```diff
@@ -1,13 +1,12 @@
-{% set kube_servers = [] -%}
-{% for fqdn in salt['mine.get']('roles:kube-master', 'fqdn', expr_form='grain').values() -%}
-  {% do kube_servers.append(fqdn) -%}
-{% endfor -%}
+{% set api_server = "api." + salt['pillar.get']('internal_infra_domain','')  -%}
```

Comment:
Not sure about this one, if we happen to fallback to the default value because the pillar value is missing, we end up with `api.`